app/views/transactions/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="header-transactions">
+  <a class="back-button" href="<%= categories_path(@category) %>">
+      <img class="back-icon" src="<%= asset_path('back.png') %>" alt="back-icon">
+  </a>
+
+  <h2 class="title-transactions">Transactions</h2>
+
+</div>
+
+<div class="individual-category-container">
+  <h2 class="category-name-individual"><%= @category.name %></h2>
+  <p class="category-amount-individual">$<%= @category.total_amount %></p>
+</div>
+
+<div class="categories">
+
+    <% @transactions.each do |transaction| %>
+
+      <div class="container-category">
+        <p class="transaction-name"><%= transaction.name %></p>
+        <p class="transaction-amount">$<%= transaction.amount %></p>
+      </div>
+      
+    <% end %>
+    </div>
+
+  <div class="add-transaction-btn-cont">
+  <a class="add-transaction-a" href="<%= new_category_transaction_path(@category) %>">
+    <button class="add-transaction-btn">Add a New Transaction</button>
+  </a>
+  </div>

```

Comment:
- [ ] Please add the correct placement of the `Add a New Transaction` button make it at the bottom of the page and give it a green color like the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/rodrigopenayo98/Budget-app-rod/assets/68796232/629dedce-3d2d-40a3-a596-edcf63d0691d)  | ![image](https://github.com/rodrigopenayo98/Budget-app-rod/assets/68796232/473edea9-6a15-486b-ad5a-917e7f0125c4)  |

Reply:
Done! ✅